src/main/kotlin/com/vanniktech/android/apk/size/ApkSizeTools.kt

```diff
@@ -0,0 +1,19 @@
+package com.vanniktech.android.apk.size
+
+import java.text.DecimalFormat
+import java.text.DecimalFormatSymbols
+import java.util.Locale
+
+object ApkSizeTools {
+  @JvmStatic fun convertBytesToMegaBytes(bytes: Long): String {
```

Comment:
let's use the shorter syntax:

`fun convertBytesToMegaBytes(bytes: Long) = getDecimalFormat().format(bytes / 1000000.toDouble())`

Reply:
Good idea.

Reply:
Fixed.